entities/license/subscription.json

```diff
@@ -62,44 +69,43 @@
       },
       "purchase_currency": {
         "type": "string",
-        "description": "The purchase currency at the moment of the subscription was created."
+        "description": "The purchase currency at the moment of the subscription was created.",
+        "enum":[
+          "USD",
+          "EUR",
+          "GBP"
+        ]
       },
       "payment_type": {
         "type": "string",
-        "description": "The payment type field defines the payment options available for a offeringת 'free' - No payment is required; 'one_time' - A single, one-time payment is required to obtain the offering; 'monthly' - A recurring payment made on a monthly basis; 'annual' - A recurring payment made on a yearly basis"
+        "description": "The payment type field defines the payment options available for a offeringת 'free' - No payment is required; 'one_time' - A single, one-time payment is required to obtain the offering; 'monthly' - A recurring payment made on a monthly basis; 'annual' - A recurring payment made on a yearly basis; 'bundle' - A subscription bundled with another subscription",
+        "enum":[
+          "free",
+          "one_time",
+          "monthly",
+          "annual",
+          "bundle"
+        ]
       },
       "bundled_from_subscription_uid": {
         "type": "string",
         "description": "The subscription uid that this subscription is bundled from"
       }
     },
     "required": [
-        "bundled_from_subscription_uid",
-        "business_uid",
-        "buyer_uid",
-        "cancellation_date",
-        "created_at",
-        "display_name",
-        "expiration_date",
-        "is_active",
-        "pay_per_unit",
-        "payment_type",
-        "directory_offering_uid",
-        "purchase_currency",
-        "purchase_price",
-        "purchase_state",
-        "subscription_uid",
-        "updated_at"
+      "directory_offering_uid",
+      "purchase_currency"
```

Comment:
You are getting this wrong. All of them are required for the entity. They are not required in the create endpoint